### ISSUE TITLE
Icinga Object "Service" rerun - failed to recreate if object_name contains spaces

### ISSUE THREAD

**Issue by opencommits** [T-Systems-MMS/ansible-collection-icinga-director#52]
Hi,
the Service creation works pretty well. Thanks for that. 

But if I run the role for a second time the role execution failed. I would expect "OK, all is fine, no changes": 

```
[ansible@bvltool01:/etc/ansible] $ ap  pb_host.yaml 
PLAY [playbook for hostnamex] **************************************************************************************

TASK [Gathering Facts] *********************************************************************************************
ok: [hostnamex.office.dir]

TASK [icinga2/director/service_new : create a service Linux Systemd httpd for host hostnamex  in icinga director] ***
fatal: [hostnamex.office.dir -> hostnamex]: FAILED! => changed=false 
  msg: 'bad return code while creating: 422. Error message: Trying to recreate icinga_service ("{"host_id":"4180","object_name":"Linux Systemd httpd"}")'

PLAY RECAP *********************************************************************************************************
hostnamex.office.dir       : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0 

```
The verbose run:
```<hostnamex> Failed to connect to the host via ssh: 
fatal: [hostnamex.office.dir -> hostnamex]: FAILED! => changed=false 
  invocation:
    module_args:
      check_command: null
      check_interval: null
      check_period: null
      check_timeout: null
      client_cert: null
      client_key: null
      disabled: false
      enable_active_checks: null
      enable_event_handler: null
      enable_notifications: null
      enable_passive_checks: null
      enable_perfdata: null
      force_basic_auth: false
      groups: []
      host: hostnamex
      imports:
      - servicetmpl_linux_systemd
      max_check_attempts: null
      notes: null
      object_name: Linux Systemd httpd
      retry_interval: null
      state: present
      url: https://monitor.example.com/icingaweb2/
      url_password: VALUE_SPECIFIED_IN_NO_LOG_PARAMETER
      url_username: director_user
      use_agent: null
      use_proxy: true
      validate_certs: true
      vars:
        systemd_unit: httpd
      volatile: null
  msg: 'bad return code while creating: 422. Error message: Trying to recreate icinga_service ("{"host_id":"4180","object_name":"Linux Systemd httpd"}")
```

Let me know if you need more log files etc. 

Thank you! 👍

**Comment by rndmh3ro** [comment]
Can you please create a minimal task to recreate this issue? Somethin like this:

```
---
- name: create service
  tags: service
  icinga_service:
    state: present
    url: "{{ icinga_url }}"
    url_username: "{{ icinga_user }}"
    url_password: "{{ icinga_pass }}"
    object_name: fooservice
    check_command: dummy
    imports:
      - fooservicetemplate
    use_agent: true
    host: foohost
    vars:
      procs_argument: consul
      procs_critical: '1:'
      procs_warning: '1:'
```

**Comment by opencommits** [comment]
Thank you for your answer.

I figer out that this Problem only occures on second run if I use " " (whitespaces) in the Object Name (`object_name: "Linux Systemd httpd"`). With `object_name: "Linux_Systemd_httpd"` all works fine.

**Comment by rndmh3ro** [comment]
This is definetly a bug we have to fix. There's probably some problem with escaping of the space. I'll have to check.

**Comment by rndmh3ro** [comment]
Fixed by #55.
@opencommits, you should now be able to use whitespaces!
